doc/sphinx/source/ops/sigmoid.rst

```diff
@@ -8,6 +8,15 @@ Sigmoid
 
    Sigmoid  // Elementwise sigmoid operation
 
+
+Description
+===========
+
+Produces a single output tensor of the same element type and shape as
+``arg,`` where the value at each coordinate of ``output`` is the
+sigmoid of ``arg`` at the same coordinate.
```

Comment:
I think we should define what we mean by "the sigmoid" here. Wikipedia, which is likely the first thing that people are going to land on when they Google, defines "a sigmoid function" as being a function with an "S-like" curve, with f(x) = 1/(1+exp(-x)) only being one example.

Reply:
I added this to the math definition.